doc/api.rst

```diff
@@ -36,26 +36,23 @@ Please refer to the :doc:`use_cases` where you can find examples of API usage.
 
   The API consists of exactly those elements described in this document, items not documented here can change release to release. Opening a `bugzilla`_ if certain needed functionality is not exposed is the right thing to do.
 
-.. _deprecating-label:
+============
+ Versioning
+============
 
-====================
- Deprecation Policy
-====================
+DNF follows the Semantic Versioning as defined at http://semver.org/.
 
-.. WARNING::
+This basically means that if your piece of software depends on e.g. DNF 1.1, the requirement can be specified as ``1.1 <= dnf < 2``. In other words, you can be sure that your software will be API-compatible with any later release of DNF until the next major version is issued. The same applies for the CLI compatibility.
 
-  The DNF project has not fully stabilized yet and it is possible parts the API described in this document can change. All such changes are documented per-version in the release notes and are subject to our deprecation policy. We do our best to reduce their number to a necessary minimum.
+.. _deprecating-label:
 
-Depracated API items (classes, methods, etc.) are designated as such in the :doc:`release_notes`. The first release where support for such items can be dropped entirely must be issued at least three months after the issue of the release that announced the deprecation and at the same time have, relatively to the deprecating release, either:
+Incompatible API changes are subject to our deprecation policy. Depracated API items (classes, methods, etc.) are designated as such in the :doc:`release_notes`. The first release where support for such items can be dropped entirely must:
 
-* a higher major version number, or
-* a higher minor version number, or
-* a patchlevel number that is *by at least three* greater.
+* be issued at least three months after the issue of the release that announced the deprecation and at the same time
```

Comment:
I would drop that. Between Fedora release and next Fedora alpha freeze we have ~2-3 months. Thats a short time and we want to push new features (incompatible) to the next release and bump the major version.